tests/python/integration/test_tuning.py

```diff
@@ -131,12 +131,11 @@ def teardown_module():
 
 
 def get_sample_task(target=tvm.target.cuda(), target_host=None):
+    target = tvm.target.Target(target, target_host)
+    target_host = target.host
```

Comment:
The `target` here is used to create a search task in the function. If `target_host` is not `None`, this would be called to update the target and host.